app/waf/src/controllers/adc.controller.ts

```diff
@@ -67,6 +67,7 @@ const prefix = '/adcaas/v1';
 export class AdcController extends BaseController {
   asgMgr: ASGManager;
   private adcStCtr: AdcStateCtrlr;
+  private logger = factory.getLogger('Unknown: controllers.adc');
```

Comment:
`private logger = factory.getLogger('Unknown: controllers.adc');` could be removed in my opinion, because we have the initialiation in the `constructor`. 
I suggest you remove L80 `{optional: true}`, because `RestBindings.Http.CONTEXT` must have been bound. 
Thus `if (reqCxt) {` is needless.